### PR TITLE
Part 1

### PR DESCRIPTION
**Pull request by ghost1497**
bloop

### CODE REVIEW

File: src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java [hunk 1]
```diff
@@ -1,7 +1,60 @@
...
+        for (String number : phoneNumber){
+            phoneNumberList.add(number);
+        }
+        phoneBook.put(name, phoneNumberList);
+    }
+
+    public String lookUp(String name){
+        String result = String.valueOf(phoneBook.get(name));
+        return result.substring(1, result.length()-1);
+    }
```
Comment (xt0fer) [inline src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java]: I guess Iwould return [0] not len()-1, but...

File: src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java [hunk 2]
```diff
@@ -1,7 +1,60 @@
...
+        return result;
+    }
+
+    public String getPhoneBook(){
+        StringBuilder result = new StringBuilder();
+        for (Map.Entry<String, ArrayList> entry : phoneBook.entrySet()){
+            result.append(String.format("%-20s %8s\n", entry.getKey(), entry.getValue()));
+        }
+        return String.valueOf(result);
+    }
```
Comment (xt0fer) [inline src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java]: this should handle the multi-number ARRLST as well.

File: src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java (hunk 1 above)
Comment (ghost1497) [reply src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java]: It prints out with brackets so it looks like "[2154637564]" and i was trying to remove the brackets

File: src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java (hunk 2 above)
Comment (ghost1497) [reply src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java]: it does. my tests were able to append out single and multi number entries.

Comment (xt0fer) [reply src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java]: yeah, but handling a clean formatting would be more to the spririt of the usage of the class.

File: src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java (hunk 1 above)
Comment (xt0fer) [reply src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java]: yeah, but handling a clean formatting would be more to the spirit of the usage of the class.

**Comment by xt0fer** [comment]
just fine.

**Pull request closed without merging**